server/src/main/scala/org/ada/server/models/WidgetSpec.scala

```diff
@@ -44,6 +44,7 @@ case class DistributionWidgetSpec(
   relativeValues: Boolean = false,
   numericBinCount: Option[Int] = None, // TODO: rename to binCount
   useDateMonthBins: Boolean = false,
+  orderByField: Option[Boolean] = None,
```

Comment:
The type `Option[Boolean]` looks ugly. Switch to `Boolean` and deliver a migration script in `db-update-0.9.1`